test/index.test.ts

```diff
@@ -333,6 +333,58 @@ describe('HipThrusTS', () => {
         // @ts-expect-error
         const returnAssignableFromCorrectShouldFail: ReturnAssignableFromCorrect = false;
       });
```

Comment:
type of pipedAToBStringToBNumber is:
const pipedAToBStringToBNumber: HasAttachData<{
    a: number;
} & Pick<{
    b: string;
}, never>, {
    b: number;
} & Pick<{
    b: string;
}, never>>

Reply:
what do you mean?